test/ocsp/helper/helper.go

```diff
@@ -323,11 +325,23 @@ func parseAndPrint(respBytes []byte, cert, issuer *x509.Certificate, config Conf
 	if resp.Certificate == nil {
 		fmt.Printf("  Certificate: nil\n")
 	} else {
-		fmt.Print("  Certificate:\n")
+		fmt.Printf("  Certificate:\n")
```

Comment:
It looks like you're matching the style of the rest of the file here, but we are typically doing the wrong thing here, without a formatting directive we should just be using `Print` rather than `Printf`.

Reply:
Hmm, this led me to do some interesting digging: in the simple case, Printf seems to be ~15% faster than Print:
```
> cat prints_test.go 
package main_test

import "fmt"
import "testing"

func BenchmarkPrint(b *testing.B) {
	for n := 0; n < b.N; n++ {
		_ = fmt.Sprint("hello")
	}
}

func BenchmarkPrintf(b *testing.B) {
	for n := 0; n < b.N; n++ {
		_ = fmt.Sprintf("hello")
	}
}

> go test -bench=. prints_test.go
goos: linux
goarch: amd64
BenchmarkPrint-8    	25311932	        47.1 ns/op
BenchmarkPrintf-8   	26826856	        40.3 ns/op
PASS
ok  	command-line-arguments	2.371s
```
I guess this makes sense, as Print is [not](https://golang.org/src/fmt/print.go?s=30292:30299#L1153) implemented in terms of Printf, as I thought it might be.

I'll update the others to consistently use Print when not using a formatter. Sigh, there goes the nice alignment :)